android/app/build.gradle

```diff
@@ -22,6 +22,9 @@ import com.android.build.OutputFile
  *   // the entry file for bundle generation
  *   entryFile: "index.android.js",
  *
+ *   // https://facebook.github.io/react-native/docs/performance#enable-the-ram-format
+ *   bundleCommand: "ram-bundle",
```

Comment:
I was curious what this was about, so I followed the link to read the docs.

Hilariously, the link is broken already. Here's a permalink for what it originally linked to:
https://reactnative.dev/docs/0.60/performance#enable-the-ram-format
and it looks like this is its new home:
https://reactnative.dev/docs/ram-bundles-inline-requires#enable-the-ram-format

(Regrettably the versioned URLs don't seem to work for the current version -- so for the current version of the docs it's impossible to create a link that both works today and will work in the future. But if you're reading this in the future, stick an `0.62/` in there.)

Let's use the updated URL. Probably that means in a future upgrade there'll be a patch that we've already taken and just conveniently drops out.

Meanwhile my conclusion from reading that bit of docs is that this won't be a priority for us to do (it requires a lot of setup and tuning to make it really a net positive), and it's obsoleted by Hermes so we'll probably be on that before we'd turn our attention to try to use this feature.

Reply:
> Probably that means in a future upgrade there'll be a patch that we've already taken and just conveniently drops out.

Ha, nope, it's still broken in master :shrug: